src/.nuget/Microsoft.NETCore.Jit/descriptions.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "Name": "RuntimePackage",
+        "Description": "Internal implementation package not meant for direct consumption.  Please do not reference directly.",
+        "CommonTypes": [ ]
+    },
+    {
+        "Name": "Microsoft.NETCore.Jit",
```

Comment:
I don't think that we want to encode the file name in the package identifier. E.g. if we end up pursuing a tiered solution, we may provide the tiers as separate JIT dlls, each with its own name. It seems better to me to tie the name to the component the package provides rather than the particular layout of the package contents.